src/shital/A12_Program3_NonRepeatingCharFromString.java

```diff
@@ -0,0 +1,40 @@
+package shital;
+
+import java.util.Scanner;
+
+class NonRepeatingCharFromString{
+	
+	
+	int findChar(String name, char ch){
+		int count =0;
+		for(int index=0;index<name.length();index++){
+			if(name.charAt(index)==ch){
+				count++;
+			}	
+		}
+		return count;
+	}
+	
+	void findNonRepeatChar(String name){
+		int value;
+		//System.out.println("Given String is: "+name);
+		for(int index=0;index<name.length();index++){
+			value=findChar(name,name.charAt(index));
+			if(value==1){
```

Comment:
here you can aslo use 
indexOf(ch) != lastIndexOf(ch) logic to avoid one more for loop